pkg/executables/kind_test.go

```diff
@@ -89,6 +89,18 @@ func TestKindCreateBootstrapClusterSuccess(t *testing.T) {
 			wantKindConfig:     "testdata/kind_config_docker_mount_networking.yaml",
 			registryMirrorTest: false,
 		},
+		{
+			name:           "With extra port mappings option",
+			wantKubeconfig: kubeConfigFile,
+			options: []testKindOption{
+				func(k *executables.Kind) bootstrapper.BootstrapClusterClientOption {
```

Comment:
How are we checking these ports have actually been mapped?

Reply:
In the unit test, I'm just checking if the generated kind config file has the expected port mapping configuration here https://github.com/aws/eks-anywhere/pull/1793/files#diff-9531b36781b70449d56a847ba73cb20fea4b287f003a2e605797e9e518c3db10R1-R23

Since we don't actually create a Kind cluster in unit tests, idk if there's another way to check if the ports actually get mapped in the Kind cluster. We will most likely have to verify this at integration test stage